### ISSUE TITLE
Document diferences between other monokai implementations

### ISSUE THREAD

**Issue by alerque** [patstockwell/vim-monokai-tasty#19]
There are a ton of other monokai implementations out there for vim already. For example I've been using [this one](https://github.com/justinmk/molokai) which includes both gui and cterm modes plus airline support.

I would suggest adding a note to the README briefly outlining any design or implementation choices that set this implementation apart from the ones that have been floating around forever.

TL;DR: What makes this tastier?

**Comment by patstockwell** [comment]
Thanks for the feedback! 😄
